diagnostic.md

```diff
@@ -7,36 +7,38 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+You use the git clone command, I used:
+git clone git@github.com:Bowers96/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+I change into the directory that I just cloned.
```

Comment:
while this is not technically incorrect, the answer we are looking for is "create a branch"